pkg/sip/inbound.go

```diff
@@ -103,8 +103,13 @@ func (s *Server) handleInviteAuth(log logger.Logger, req *sip.Request, tx sip.Se
 		_ = tx.Respond(sip.NewResponseFromRequest(req, 100, "Processing", nil))
 	}
 
-	inviteState := s.getInvite(from)
-	log = log.WithValues("inviteStateFrom", from)
+	// Extract SIP Call ID for tracking in-progress invites
+	sipCallID := ""
+	if h := req.CallID(); h != nil {
```

Comment:
We should require CallID parameter the same way as we do with From header.

Reply:
I actually thought about that. But I was wondering if we should do that at the sipgo level(if it isn't already being done) when parsing the invite. Invite without sipcallID won't be valid at all right ?